### ISSUE TITLE
[chart/redis-ha]support configuration for redis container command parameter

### ISSUE THREAD

**Issue by GDegrove** [DandyDeveloper/charts#254]
**Is your feature request related to a problem? Please describe.**

I'm trying to deploy redis-ha using redis-stack-server as the image. 

From what I can see, the current helm chart only start the container with redis-server. However, when using the redis-stack-server you want to start the contianer with `/entrypoint.sh` (don't know why the startup script is name entrypoint though...) or using the `redis-stack-server`. 
 
**Describe the solution you'd like**

Add a configuration for the helm chart to contain a way to override/configure the command. Instead of using the hard-coded `edis-server`, we may want to let user chose. 
This would allow a developer to easily change the configuration of the helm chart without spending too much time on making sure their image is compatible with it.

**Describe alternatives you've considered**

- make a change in the upstream image
- download module with an init container

**Comment by DandyDeveloper** [comment]
Yes. #238 would do this, it was neglected. I'll re-review to see if this is definitely fit for purpose and try to get this merged in.
